docs/index.md

```diff
@@ -9,16 +9,16 @@
 
 ROCm is an open-source software platform optimized to extract HPC and AI workload
 performance from AMD Instinct accelerators and AMD Radeon GPUs while maintaining
-compatibility with industry software frameworks. For more information, see [What is ROCm?](./what-is-rocm.rst)
+compatibility with industry software frameworks. For more information, see
+[What is ROCm?](./what-is-rocm.rst)
 
-If you're using Radeon GPUs, consider reviewing {doc}`Radeon-specific ROCm documentation<radeon:index>`.
-
-Installation instructions are available from:
+The ROCm platform uses the Heterogeneous System Architecture (HSA) to support
+computing across multiple hardware architectures, letting developers write code
+that runs on both AMD CPUs and GPUs. ROCm supports multiple programming
+languages such as HIP (Heterogeneous-Compute Interface for Portability), OpenCL,
+and OpenMP, as described in {doc}`HIP programming guide<hip:index>`.
```

Comment:
The hip programming guide (the link actually goes to the landing page) is only relevant to hip.
```suggestion
languages such as {doc}`HIP (Heterogeneous-Compute Interface for Portability)<hip:index>`, OpenCL,
and OpenMP.
```